Nucleon/README.md

```diff
@@ -0,0 +1,16 @@
+
+# Nucleon
+
+Nucleon learns about the external threat landscape in order to determine how threats can impact your security posture. Nucleon delivers notifications for pre-attack adversary behaviour, empowering you to truly protect your network. 
+
+As part of its core functions, the Nucleon polymorphic sensors identify impending threats before impact, automating these early-warnings effectively. With our active threat feeds and robust client integrations, these all equal early threat detection and visibility, and increased protection.  
+
+An additional set of these polymorphic sensors sit closer to the gateways of your network, collecting data about sensitive and vulnerable access points that are unique to your network specifically. They expand your attack surface in order to lure in and potentially trap those bad actors. 
+
+We provide only actionable alerts, actively guiding you to block the attack source with the help of your existing defense systems. From the Nucleon dashboard, you can access threat intelligence assessments, threat alerts, insights and remediation guidance that contribute to a strengthened security posture. 
+
+The process is described in detail. For any questions or concerns please contact us through your Slack channel or at Support@nucleon.sh
+
+# Documentation
+  <Provide app docs link>
```

Comment:
Do you have the doc link or will you update it in another PR? You can put the link here if it's still not live yet.

Reply:
we don't have it yet, we will update it in another PR